src/tests/ftest/recovery/pass_1.py

```diff
@@ -187,32 +220,20 @@ def test_orphan_pool_trus_ps(self):
         :avocado: tags=Pass1Test,test_orphan_pool_trust_ps
         """
         errors = []
-        # Run step 1 to 6.
+        # 4. Run DAOS checker under kinds of mode.
```

Comment:
Not sure if it's a good idea to remove steps from the method description. I know it's duplicate of what you wrote in `run_checker_on_orphan_pool()`, but it would make it harder to understand what this test does. If you really want to remove it, then you might want to improve the comment here. e.g.,
```suggestion
        # 1-4. Create a pool, remove the PS entry, list and verify that there's no pool, run and verify the checker behavior.
```

Reply:
The steps are moved to run_checker, not removed.